nimble/include/nimble/ble.h

```diff
@@ -177,8 +177,8 @@ struct ble_mbuf_hdr
 #define BLE_MBUF_MEMBLOCK_OVERHEAD      \
     (sizeof(struct os_mbuf) + BLE_MBUF_PKTHDR_OVERHEAD)
 
-/* Length of host user header.  Only contains the peer's connection handle. */
-#define BLE_MBUF_HS_HDR_LEN     (2)
+#define BLE_MBUF_HS_HDR_LEN     (0)
```

Comment:
This comment is definitely in "nitpick territory" so feel free to ignore it.  IMO, this constant reduces clarity without any benefit.  It would be more consistent and easily understood to just hardcode 0 at the allocation site.

Reply:
I tend to agree, although the reason I left this constant was just in case we want to restore pkthdr later. But since it was removed because it does not quite work as expected on host side, I do not think this will happen any time soon :-)